entity-framework/core/querying/raw-sql.md

```diff
@@ -10,16 +10,16 @@ uid: core/querying/raw-sql
 Entity Framework Core allows you to drop down to raw SQL queries when working with a relational database. This can be useful if the query you want to perform can't be expressed using LINQ, or if using a LINQ query is resulting in inefficient SQL queries. Raw SQL queries can return entity types or, starting with EF Core 2.1, [query types](xref:core/modeling/query-types) that are part of your model.
```

Comment:
Will hold off until the query types section itself is modified to keyless entity type (link has to be updated).

Reply:
The PR that renames query types to keyless entity types is out.
```suggestion
Entity Framework Core allows you to drop down to raw SQL queries when working with a relational database. This can be useful if the query you want to perform can't be expressed using LINQ, or if using a LINQ query is resulting in inefficient SQL queries. Raw SQL queries can be used to return any entities that are part of your model, including regular entities and [keyless entitties](xref:core/modeling/query-types).
```

Reply:
The page also got renamed.